ui.frontend/src/utils.js

```diff
@@ -326,6 +359,10 @@ class Utils {
                     _element: elements[i]
                 });
                 Utils.initializeAllFields(formContainer);
+                const autoSaveProperties = _formJson.properties?.['fd:autoSave'];
+                if (window.Granite && !window.Granite.author && autoSaveProperties) {
```

Comment:
You shouldn't need Granite namespace in runtime

Reply:
Done